### PR TITLE
Update 03-generators.mdx

### PR DESCRIPTION
**Pull request by nikolasburk**
Two main changes:

- remove Prismaliser and prismix since none of them are actually generators that are defined in the Prisma schema via a `generator` block
- add `zod-prisma` and `prisma-giraphql-types ` generators

### CODE REVIEW

**Review by tanberry — APPROVED** [review]
Thanks @nikolasburk ! Only tweak I see is to add a hyphen in "Prisma-based", otherwise looks great.

**Review by janpio — APPROVED** [review]

**Comment by nikolasburk** [comment]
@tanberry hmm weird, aren't you able to see these changes in the file?

image

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 1]
```diff
@@ -73,6 +73,6 @@ The following is a list of community created generators.
...
-- [`Prismaliser`](https://github.com/Ovyerus/prismaliser): Prismaliser is a visualisation webapp for Prisma schemas. It allows you to visually explore your schema and the relations between your models, by showing links between the different types of relations in the schema (many-to-many, one-to-many, one-to-one), similar to an Entity-relationship model.
 - [`prisma-class-generator`](https://github.com/kimjbstar/prisma-class-generator): Generates classes from your Prisma Schema that can be used as DTO, Swagger Response, TypeGraphQL and so on.
+- [`zod-prisma`](https://github.com/CarterGrimmeisen/zod-prisma): Creates Zod schemas from your Prisma models.
+- [`prisma-giraphql-types`](https://github.com/hayes/giraphql/tree/main/packages/plugin-prisma): Makes it easier to define Prisma based object types, and helps solve n+1 queries for relations. It also has integrations for the Relay plugin to make defining nodes and connections easy and efficient.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]: I think this was meant:
```suggestion
- [`prisma-giraphql-types`](https://github.com/hayes/giraphql/tree/main/packages/plugin-prisma): Makes it easier to define Prisma-based object types, and helps solve n+1 queries for relations. It also has integrations for the Relay plugin to make defining nodes and connections easy and efficient.
```

**Comment by nikolasburk** [comment]
(sorry I realized I completely misunderstood your message, so deleted my comment from earlier 😅  @ Tana)

**Comment by tanberry** [comment]
Thanks again @nikolasburk for keeping our docs up-to-date!

**Pull request merged**